cachito/workers/pkg_managers/general_js.py

```diff
@@ -133,8 +169,22 @@ def download_dependencies(request_id, deps, proxy_repo_url, skip_deps=None):
             for dep_pair in list(zip(output.split("\n"), dep_batch)):
                 tarball = dep_pair[0]  # e.g. angular-animations-8.2.0.tgz
                 dir_path = dep_pair[1].rsplit("@", 1)[0]  # e.g. @angular/animations
+                version = dep_pair[1].rsplit("@", 1)[1]
```

Comment:
We could split it only once here with `dir_path, version = dep_pair[1].rsplit("@", 1)`

Reply:
While it's a one-liner and a good suggestion, I'd prefer to keep it as it is just for the sake of readability and having an example there in-line can help make things easier to understand IMO. lmk what do you think.

Reply:
This is outdated. Resolving